R/linear_pool.R

```diff
@@ -74,16 +76,22 @@
 #' all.equal(lp_from_component_qs$value, expected_quantiles, tolerance = 1e-2,
 #'           check.attributes = FALSE)
 #'
-
+#' @importFrom lifecycle deprecated
 linear_pool <- function(model_out_tbl, weights = NULL,
                         weights_col_name = "weight",
                         model_id = "hub-ensemble",
                         task_id_cols = NULL,
                         compound_taskid_set = NA,
-                        derived_tasks = NULL,
+                        derived_task_ids = NULL,
+                        derived_tasks = lifecycle::deprecated(),
                         n_samples = 1e4,
                         n_output_samples = NULL,
                         ...) {
```

Comment:
I believe this will still work, but I think it would be better to place the deprecated argument at the end of the function call. This way people who use positional arguments don't get a nasty surprise.

```suggestion
                        derived_task_ids = NULL,
                        n_samples = 1e4,
                        n_output_samples = NULL,
                        ...,
                        derived_tasks = lifecycle::deprecated()) {
```